modules/ROOT/pages/running-containers.adoc

```diff
@@ -69,4 +69,4 @@ systemd:
 ----
 
 === For more information
-See the https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/container.md#docker[etcd documentation] for more information on running etcd in containers and how to set up multi-node etcd.
+See the https://etcd.io/docs/v3.4/op-guide/container/#docker[etcd documentation] for more information on running etcd in containers and how to set up multi-node etcd.
```

Comment:
Is there an uversionsed version of these docs (i.e. latest). If there is, should we use that instead?

Reply:
In fact, there is.  Updated.